src/documents/tests/utils.py

```diff
@@ -426,3 +428,9 @@ def send_progress(
             payload["data"].update(extra_args)
 
         self.payloads.append(payload)
+
+
+def pseudo_random_text(length=100, seed=42):
```

Comment:
What do you think about [faker.text](https://faker.readthedocs.io/en/stable/providers/faker.providers.lorem.html#faker.providers.lorem.Provider.text) instead?  It is an "approximate number of characters", so this would need to check and maybe add more.  Or perhaps double the amount, then limit

Reply:
I thought it wasn't in the project requirements. I can do it with `faker`, for sure.

Reply:
The test looks cleaner now, thank you!